xamarin/Xamarin.Forms
# [Android] Shapes in TitleView

### Description of Change ###

In 5.0 it works in Android and iOS. This PR only include a sample to validate the issue.

### Issues Resolved ### 

- fixes #12609 

### API Changes ###

 None

### Platforms Affected ### 

- Android


### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

Not applicable

### Testing Procedure ###
Launch Core Gallery and navigate to the issue 12609. If you can see the figure in the TitleBar, the test has passed.

### PR Checklist ###
